src/xcode/ENA/ENA/Source/Scenes/Events/Store/Model/Checkin.swift

```diff
@@ -7,16 +7,17 @@ import Foundation
 struct Checkin: Equatable {
 
 	let id: Int
-	let traceLocationGUID: String
-	let traceLocationGUIDHash: Data
+	let traceLocationId: Data
+	let traceLocationIdHash: Data
 	let traceLocationVersion: Int
 	let traceLocationType: TraceLocationType
 	let traceLocationDescription: String
 	let traceLocationAddress: String
 	let traceLocationStartDate: Date?
 	let traceLocationEndDate: Date?
 	let traceLocationDefaultCheckInLengthInMinutes: Int?
-	let traceLocationSignature: String
+	let cryptographicSeed: Data
+	let cnMainPublicKey: Data
```

Comment:
Any form of documentation would help a lot. At least a link to the tech spec or somebody will ask why we have a "chinese main public key" in the app.

Reply:
🤷‍♂️ Naming from the spec.

Reply:
I'll add a link to the spec